stable/grafana/README.md

```diff
@@ -53,6 +53,7 @@ You have to add --force to your helm upgrade command as the labels of the chart
 | `priorityClassName`                       | Name of Priority Class to assign pods         | `nil`                                                   |
 | `image.repository`                        | Image repository                              | `grafana/grafana`                                       |
 | `image.tag`                               | Image tag (`Must be >= 5.0.0`)                | `7.0.3`                                                 |
+| `image.sha`                               | Image sha (optional)                          | `17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe` |
```

Comment:
this isn't really optional if it's being set a default value

Reply:
It’s optional because it won’t fail if you don’t provide the value. This makes it backwards compatible to users who are wrapping this chart and don’t have SHA in their values.

The plan is to offer security by default.

Reply:
when users who are using a different image tag than 7.0.3 update to this chart version they'd had issues though

Reply:
actually im mixing up digests and sha

Reply:
no actually on my workstation it always pulls the SHA rather than the tag:

```
❯ docker pull grafana/grafana:7.0.5@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe: Pulling from grafana/grafana
Digest: sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
Status: Image is up to date for grafana/grafana@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
docker.io/grafana/grafana:7.0.5@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
❯ docker pull grafana/grafana:7.1.0@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe: Pulling from grafana/grafana
Digest: sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
Status: Image is up to date for grafana/grafana@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
docker.io/grafana/grafana:7.1.0@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
❯ docker pull grafana/grafana:7.0.0@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe: Pulling from grafana/grafana
Digest: sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
Status: Image is up to date for grafana/grafana@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
docker.io/grafana/grafana:7.0.0@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
❯ docker pull grafana/grafana:9999.999.999@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe: Pulling from grafana/grafana
Digest: sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
Status: Image is up to date for grafana/grafana@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
docker.io/grafana/grafana:9999.999.999@sha256:17cbd08b9515fda889ca959e9d72ee6f3327c8f1844a3336dfd952134f38e2fe
```

Reply:
@calvinbui I see what you mean,. I'm sending another PR without default values.

Reply:
https://github.com/helm/charts/pull/23240